Prace_domowe/Praca_domowa3/Grupa1/Kacper_Grzymkowski/PD3.ipynb

```diff
@@ -0,0 +1,870 @@
+{
```

Comment:
<p>Dobrze, że odniesiono się do tego, że klasy nie są zbalansowane i jak to wpływa na wybór metryki</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>